specs/experimental/op-contracts-manager.md

```diff
@@ -18,50 +18,90 @@ of governance approved [contract releases] can be found on the
   - [Chain ID Frontrunning](#chain-id-frontrunning)
   - [Chain ID Value](#chain-id-value)
   - [Proxy Admin Owner](#proxy-admin-owner)
-  - [Upgradeability (ABI Changes)](#upgradeability-abi-changes)
+  - [Safely using `DELEGATECALL`](#safely-using-delegatecall)
+  - [Atomicity of upgrades](#atomicity-of-upgrades)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Deployment
+## Overview
+
+The OP Contracts Manager refers to a series of contracts, of which a new singleton is deployed
+for each new release of the OP Stack contracts.
 
-The OP Contracts Manager is a proxied contract deployed at `0xTODO`. It can be deployed as follows:
+The OP Contracts Manager corresponding to each release can be used to:
 
-TODO.
+1. Deploy a new OP chain.
+2. Upgrade the contracts for an existing OP chain from the previous release to the new release.
+3. Orchestrate adding a new game type on a per-chain basis
+
+## Getter Methods
+
+The following interface defines the available getter methods:
+
+```solidity
+/// @notice The logic address and initializer selector for an implementation contract.
+struct Implementation {
+  address logic; // Address containing the deployed logic contract.
+  bytes4 initializer; // Function selector for the initializer.
+}
+
+/// @notice Returns the latest approved release of the OP Stack contracts.
+/// @notice Release strings follow semver and are named with the
+/// format `op-contracts/vX.Y.Z`.
+function latestRelease() external view returns (string memory);
+
+/// @notice Returns the implementation contract addresses.
+function implementations() public view returns (Implementations memory);
+
+/// @notice Maps an L2 Chain ID to the SystemConfig address for that chain.
+/// @notice All contracts for a chain can be found from its SystemConfig.
+function systemConfigs(uint256 chainId) external view returns (SystemConfig);
 
-## Interface
+/// @notice Maps an L2 chain ID to an L1 batch inbox address as defined by the standard
+function chainIdToBatchInboxAddress(uint256 _l2ChainId) public pure returns (address);
 
-Version 1.0.0 of the OP Contracts Manager deploys the `op-contracts/v1.6.0`
-contracts release.
+/// @notice Returns the blueprint contract addresses.
+function blueprints() public view returns (Blueprints memory);
+```
 
-### `Proxy.sol`
+## Deployment
 
-The OP Contracts Manager is a proxied contract using the standard `Proxy.sol` contract that lives in
-the Optimism monorepo. Therefore the OP Contracts Manager will have the same interface as the
-`Proxy.sol`, in addition to other methods defined in this specification.
+### Interface
 
-The privileged methods of the OP Contracts Manager will be held by the L1 ProxyAdmin owner, as
-specified by the [standard configuration].
+Version 1.0.0 of the OP Contracts Manager deploys the `op-contracts/v1.6.0` contracts release,
+and is deployed at `0x9BC0A1eD534BFb31a6Be69e5b767Cba332f14347`. In the future this will
```

Comment:
I think this shouldn't be called version 1.0.0 of OPCM anymore right? The version of OPCM is tied to the version of the l1 smart contract release that it deploys. 

Also, I think it would be helpful to link to the OPCM addresses stored in op-deployer [here](https://github.com/ethereum-optimism/optimism/blob/564b0c17198bff7c3eebc7c328e1bf57b9a46188/op-deployer/pkg/deployer/standard/standard.go#L202-L217). We've yet to properly added these to superchain-registry and ensure they're kept up to date. 

In fact, the addresses in scr are currently wrong,
- [mainnet](https://github.com/ethereum-optimism/superchain-registry/blob/a54d203e727f1cdeff06b883e239febf92b4b1d6/superchain/configs/mainnet/superchain.toml#L4)
- [sepolia](https://github.com/ethereum-optimism/superchain-registry/blob/a54d203e727f1cdeff06b883e239febf92b4b1d6/superchain/configs/sepolia/superchain.toml#L4)

Unrelated, but we're going to need to find a way to properly store these in superchain-registry.

Reply:
> I think this shouldn't be called version 1.0.0 of OPCM anymore right? The version of OPCM is tied to the version of the l1 smart contract release that it deploys.

I may have previously written or said otherwise somewhere, but I do think the OPCM should have both its own `version()` and `l1ContractsRelease()` as independent versions.
Otherwise we'll increment the OPCM version much less frequently, and since the OPCM is its own contract which can vary independently of changes made to other L1 contracts, it feels conservative to be consistent with the practice used in other contracts.

Reply:
We agreed we won't make any changes to the current `version()` approach.